supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,70 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location. 
+* Avoid irrelevant link text.
+* You can use a sentence or sentence fragment as the link text.
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within www.redhat.com.
+
+.Example AsciiDoc: Internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to another guide or to a user story within the same guide:
+----
+For more information about <topic>, see link:<link>[<link_text>] in _<title_name>_.
+----
+
+[[rh-kb-links]]
+=== Linking to Red Hat Knowledgebase articles
+
+* Use the title of the Knowledgebase article for the link text, or use descriptive running text.
+* Call out that this is a Knowledgebase article when _not_ using running text.
+* When the link appears in *Additional resources*, put the article title first, followed by `(Red Hat Knowledgebase)`.
+
+
+.Example: Article title
+
+For a non-cloud environment, you can resize the disk and file system. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/199573[Does RHEL 7 support online resize of disk partitions?].
+
+.Example: Running text
+
+If your Apache web server configuration enables SSL security, verify that you enable only the TLSv1 protocol and disable SSLv2 and SSLv3. This is because of the link:https://access.redhat.com/solutions/1232413[POODLE SSL vulnerability (CVE-2014-3566)].
+
+.Example: Additional resources
+
+* link:https://access.redhat.com/solutions/199573[Does RHEL 7 support online resize of disk partitions?] (Red Hat Knowledgebase)
+
+
+
 [[external-links]]
 == External links
+
+External links are links to a domain other than www.redhat.com. Include links to external sites only when necessary. External links can change or be unreliable.
```

Comment:
Mention wider project support on providing an external link in docs?

Reply:
Hey @dfitzmau what d'you mean exactly? :)

Reply:
Hi @ndeevy . My apologies, I should have padded my question out a bit more. 

Would adding a note that specifies that one should seek agreement from a product's overall RH team on provided links in RH docs that point to external sources?

Reply:
Oops forgot to reply here @dfitzmau 
I think we can give this general guidance to writers to use links with caution and care but leave it to individual product teams to then decide if a link is necessary or not. If more folks think we need a note here, I'm all ears and open to a draft of it too.